### ISSUE TITLE
Rate your experience feedback mechanism.

### ISSUE THREAD

**Issue by ATMiddleton** [department-of-veterans-affairs/va.gov-team#65148]
## Issue Description
Create and/or Update Feedback survey designs
---
## Tasks
-Draft: Research Plan for Search & Discovery Team, Resources and Support, Submit Feedback Tool 2.0
Background
Resources and support (R&S) contains tier 2 content (benefit-adjacent content). The Submit Feedback Tool was introduced to R&S article pages last year with the option to rate articles as either good or bad. As a part of the Submit Feedback Tool 2.0 product initiative, once a user rates an article as 'bad', we would like to implement a way for users to report the reason why they rated the page poorly. We would also like these issues to be actionable and easily accessible for the content owners so that improvements can be easily made.

We would like to make improvements to the layout of R&S article pages.

Discovery: Resources and Support Article Pages

Resources and Support Roadmap Session Summary

Learning Center Original Reseach Folder

Learning Center Moderated Usability Study
[Link to product brief coming soon]

OCTO Objectives
Improving Resources and Support will allow Veterans and those in their support system to find information on utilizing their VA.gov account, gaining supplemental information on benefits, and easily find links to apply for benefits or make changes to their account.

Veteran Journey
Resources and support impacts the Veteran journey from Starting up and all phases thereafter.

Research Goals and Questions
1. Validate that users are able to easily use the Submit Feedback Tool to rate the article page and report an appropriate problem.
Are users able to find where they can report feedback?
What do users expect to see after they submit feedback?
In what situation would a user rate this page? Why would they rate it 'good', why would they rate it 'bad'?
Does the wording make sense? If not, what should it say?
Do the options under "How would you rate your experience' make sense to users?
What other types of problems would they expect to report?
What do users think about the amount of issues presented to them? Is the amount overwhelming?
What are their perceptions on the difference between this feedback tool and Medallia?
Content creator question:
What type of feedback is helpful and actionable to them?
How can we make this feedback more accessible to them?
2. Are users able to easily navigate and discover the rate your experience feedback mechanism on the article pages?
Do users utilize rate your experience feedback mechanism on the article pages?
Do users utilize the feedback button near the footer on the article pages?
3. Is the question "How would you rate your experience" a helpful title that tells users what they will find?
What do users think of the title, "How would you rate your experience"?
If users do not feel like "How would you rate your experience" reflects what they find on the section, what else would they name it?
Outcome
We will confirm whether:

Updates allow users to provide feedback which will inform content editors on changes that can be made to improve R&S articles
Updates to R&S article pages improve navigation and discoverability of more R&S articles
The name "Resources and support" accurately informs users of what content they will find
Hypothesis
Users will not know the purpose of the rate your experience feedback mechanism,
The good/bad buttons that are currently used to track feedback, are not helpful as the data is recorded in Google Analytics.
Users will be more likely to use the feedback button near the footer to leave feedback on the Article page
Method
We'll conduct remote usability testing over Zoom using a prototype.

Location
Zoom

Research materials
For moderated usability tests:

[Link to conversation guide](url goes here)
[Link to prototype](url goes here)
Recruitment
Recruitment approach
We will recruit Veteran participants using a lean maximum variation strategy. We'll leverage Perigean's recruiting services to find our participants.

Recruitment criteria
Schedule 10 Veterans for a minimum of 8 successfully completed sessions.

Primary criteria (must-haves)

50% of participants should be able to participant from a desktop device.

50% of participants should be able to participate from a mobile phone.

Secondary criteria (nice-to-haves)

Gender

8 women, 8 men
Inclusion

3 participants who use assistive technology (e.g. screen reader, magnification)
10 participants who identify as other than white
5 participants who identify as other than straight/heterosexual and cisgender
8 participants age 55+
8 participants who identify as having a cognitive disability
5 participants who do not have a college degree
5 participants who live in a rural area
Timeline
If you are using Perigean to recruit please submit 1 FULL week prior to the start of research for remote research, 2+ weeks for in person.

Prepare
When will the thing you are testing be finalized? (Goes without saying, but should be a few days before testing will begin.)

Please indicate the date and name of a mock participant for a pilot session.

Pilot participant email:
Date and time of pilot session:
Research sessions
What dates do you plan to do research?

Length of sessions
45 minutes. Perigean will schedule the sessions with buffer 15 minutes of buffer time to allow for participants who can't make it on time, or if you might go over time.

Availability
When would you like sessions scheduled? Please list exact dates and times in EASTERN Standard Time. Note: we recommend providing availability outside of work hours, as many Veterans are only available before and after working times, and live across the U.S.
Please request enough dates and time slots for the number of requested participants. (e.g. Monday 9-1, 3-6; Tuesday 9-6, etc.).

Team Roles
Moderator: Camille Green ,404-428-2313,Camille.green@oddball.io
Research guide writing and task development: Camille Green
Participant recruiting & screening: Perigean
Project point of contact: Camille Green
Participant(s) for pilot test: TBD
Note-takers: Perigean, Aubrey Archangel, Sarifa Khalilullah
Observers: chante.lantosswett@va.gov,anita.middleton@oddball.io

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by UXCamG** [comment]
High fidelity mockups and Userflow:  https://sketch.com/s/dbc369e6-21ae-4717-a169-a5e734934f93

**Comment by UXCamG** [comment]
# Draft: Research Plan for Search & Discovery Team, Resources and Support, Submit Feedback Tool 2.0

## Background
Resources and support (R&S) contains tier 2 content (benefit-adjacent content). The Submit Feedback Tool was introduced to R&S article pages last year with the option to rate articles as either good or bad. As a part of the Submit Feedback Tool 2.0 product initiative, once a user rates an article as 'bad', we would like to implement a way for users to report the reason why they rated the page poorly. We would also like these issues to be actionable and easily accessible for the content owners so that improvements can be easily made.

We would like to make improvements to the layout of R&S article pages. 
- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/resources-and-support/article-pages/discovery.md">Discovery: Resources and Support Article Pages</a>

- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/issues/32746#issuecomment-970466533">Resources and Support Roadmap Session Summary</a>

- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/tree/69833737d9fe22b8990bb987e7c50de13205c5d5/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research">Learning Center Original Reseach Folder</a>
	- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/learning-center-mvp-findings-summary.md">Learning Center Moderated Usability Study</a>

[Link to product brief coming soon]

### OCTO Objectives 
Improving Resources and Support will allow Veterans and those in their support system to find information on utilizing their VA.gov account, gaining supplemental information on benefits, and easily find links to apply for benefits or make changes to their account.

### Veteran Journey
Resources and support impacts the Veteran journey from Starting up and all phases thereafter.

## Research Goals and Questions	

### 1. Validate that users are able to easily use the Submit Feedback Tool to rate the article page and report an appropriate problem.
- Are users able to find where they can report feedback?
- What do users expect to see after they submit feedback?
- In what situation would a user rate this page? Why would they rate it 'good', why would they rate it 'bad'?
- Does the wording make sense? If not, what should it say?
- Do the options under "How would you rate your experience' make sense to users?
- What other types of problems would they expect to report?
- What do users think about the amount of issues presented to them? Is the amount overwhelming?
- What are their perceptions on the difference between this feedback tool and Medallia?
- **Content creator question:** 
	- What type of feedback is helpful and actionable to them?
	- How can we make this feedback more accessible to them?

### 2. Are users able to easily navigate and discover the rate your experience feedback mechanism on the article pages?
- Do users utilize rate your experience feedback mechanism on the article pages?
- Do users utilize the feedback button near the footer on the article pages?


### 3. Is the question "How would you rate your experience" a helpful title that tells users what they will find?
- What do users think of the title, "How would you rate your experience"?
- If users do not feel like "How would you rate your experience" reflects what they find on the section, what else would they name it?


### Outcome
We will confirm whether:
- Updates allow users to provide feedback which will inform content editors on changes that can be made to improve R&S articles
- Updates to R&S article pages improve navigation and discoverability of more R&S articles
- The name "Resources and support" accurately informs users of what content they will find


### Hypothesis

- Users will not know the purpose of the rate your experience feedback mechanism, 
- The good/bad buttons that are currently used to track feedback, are not helpful as the data is recorded in Google Analytics. 
- Users will be more likely to use the feedback button near the footer to leave feedback on the Article page 

## Method	
We'll conduct remote usability testing over Zoom using a prototype.

### Location
Zoom

### Research materials

For moderated usability tests: 
- [Link to conversation guide](url goes here)
- [Link to prototype](url goes here)

	
## Recruitment	

### Recruitment approach
We will recruit Veteran participants using a lean maximum variation strategy. We'll leverage Perigean's recruiting services to find our participants.

### Recruitment criteria
Schedule 10 Veterans for a minimum of 8 successfully completed sessions.

**Primary criteria (must-haves)**

- 50% of participants should be able to participant from a desktop device.

- 50% of participants should be able to participate from a mobile phone.



**Secondary criteria (nice-to-haves)**

**Gender**
- 8 women, 8 men

**Inclusion**
- 3 participants who use assistive technology (e.g. screen reader, magnification)
- 10 participants who identify as other than white
- 5 participants who identify as other than straight/heterosexual and cisgender
- 8 participants age 55+
- 8 participants who identify as having a cognitive disability
- 5 participants who do not have a college degree
- 5 participants who live in a rural area


## Timeline
If you are using Perigean to recruit please submit 1 FULL week prior to the start of research for remote research, 2+ weeks for in person. 

### Prepare
When will the thing you are testing be finalized? (Goes without saying, but should be a few days before testing will begin.)

Please indicate the date and name of a mock participant for a pilot session. 
* Pilot participant email:
* Date and time of pilot session: 

### Research sessions
What dates do you plan to do research? 

### Length of sessions
45 minutes. Perigean will schedule the sessions with buffer 15 minutes of buffer time to allow for participants who can't make it on time, or if you might go over time.

### Availability
When would you like sessions scheduled? Please list exact dates and times in EASTERN Standard Time. *Note: we recommend providing availability outside of work hours, as many Veterans are only available before and after working times, and live across the U.S.* 
Please request enough dates and time slots for the number of requested participants. (e.g. Monday 9-1, 3-6; Tuesday 9-6, etc.). 
	
## Team Roles	

- Moderator: Camille Green ,404-428-2313,Camille.green@oddball.io
- Research guide writing and task development: Camille Green
- Participant recruiting & screening: Perigean	
- Project point of contact: Camille Green
- Participant(s) for pilot test: TBD	
- Note-takers: Perigean, Aubrey Archangel, Sarifa Khalilullah 
- Observers: chante.lantosswett@va.gov,anita.middleton@oddball.io

**Comment by UXCamG** [comment]
# Draft: Research Plan for Search & Discovery Team, Resources and Support, Submit Feedback Tool 2.0

## Background
Resources and support (R&S) contains tier 2 content (benefit-adjacent content). The Submit Feedback Tool was introduced to R&S article pages last year with the option to rate articles as either good or bad. As a part of the Submit Feedback Tool 2.0 product initiative, once a user rates an article as 'bad', we would like to implement a way for users to report the reason why they rated the page poorly. We would also like these issues to be actionable and easily accessible for the content owners so that improvements can be easily made.

We would like to make improvements to the layout of R&S article pages. 
- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/resources-and-support/article-pages/discovery.md">Discovery: Resources and Support Article Pages</a>

- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/issues/32746#issuecomment-970466533">Resources and Support Roadmap Session Summary</a>

- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/tree/69833737d9fe22b8990bb987e7c50de13205c5d5/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research">Learning Center Original Reseach Folder</a>
	- <a href="https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/learning-center-mvp-findings-summary.md">Learning Center Moderated Usability Study</a>

[Link to product brief coming soon]

### OCTO Objectives 
Improving Resources and Support will allow Veterans and those in their support system to find information on utilizing their VA.gov account, gaining supplemental information on benefits, and easily find links to apply for benefits or make changes to their account.

### Veteran Journey
Resources and support impacts the Veteran journey from Starting up and all phases thereafter.

## Research Goals and Questions	

### 1. Validate that users are able to easily use the Submit Feedback Tool to rate the article page and report an appropriate problem.
- Are users able to find where they can report feedback?
- What do users expect to see after they submit feedback?
- In what situation would a user rate this page? Why would they rate it 'good', why would they rate it 'bad'?
- Does the wording make sense? If not, what should it say?
- Do the options under "How would you rate your experience' make sense to users?
- What other types of problems would they expect to report?
- What do users think about the amount of issues presented to them? Is the amount overwhelming?
- What are their perceptions on the difference between this feedback tool and Medallia?
- **Content creator question:** 
	- What type of feedback is helpful and actionable to them?
	- How can we make this feedback more accessible to them?

### 2. Are users able to easily navigate and discover the rate your experience feedback mechanism on the article pages?
- Do users utilize rate your experience feedback mechanism on the article pages?
- Do users utilize the feedback button near the footer on the article pages?


### 3. Is the question "How would you rate your experience" a helpful title that tells users what they will find?
- What do users think of the title, "How would you rate your experience"?
- If users do not feel like "How would you rate your experience" reflects what they find on the section, what else would they name it?


### Outcome
We will confirm whether:
- Updates allow users to provide feedback which will inform content editors on changes that can be made to improve R&S articles
- Updates to R&S article pages improve navigation and discoverability of more R&S articles
- The name "Resources and support" accurately informs users of what content they will find


### Hypothesis

- Users will not know the purpose of the rate your experience feedback mechanism, 
- The good/bad buttons that are currently used to track feedback, are not helpful as the data is recorded in Google Analytics. 
- Users will be more likely to use the feedback button near the footer to leave feedback on the Article page 

## Method	
We'll conduct remote usability testing over Zoom using a prototype.

### Location
Zoom

### Research materials

For moderated usability tests: 
- [Link to conversation guide](url goes here)
- [Link to prototype](url goes here)

	
## Recruitment	

### Recruitment approach
We will recruit Veteran participants using a lean maximum variation strategy. We'll leverage Perigean's recruiting services to find our participants.

### Recruitment criteria
Schedule 10 Veterans for a minimum of 8 successfully completed sessions.

**Primary criteria (must-haves)**

- 50% of participants should be able to participant from a desktop device.

- 50% of participants should be able to participate from a mobile phone.



**Secondary criteria (nice-to-haves)**

**Gender**
- 8 women, 8 men

**Inclusion**
- 3 participants who use assistive technology (e.g. screen reader, magnification)
- 10 participants who identify as other than white
- 5 participants who identify as other than straight/heterosexual and cisgender
- 8 participants age 55+
- 8 participants who identify as having a cognitive disability
- 5 participants who do not have a college degree
- 5 participants who live in a rural area


## Timeline
If you are using Perigean to recruit please submit 1 FULL week prior to the start of research for remote research, 2+ weeks for in person. 

### Prepare
When will the thing you are testing be finalized? (Goes without saying, but should be a few days before testing will begin.)

Please indicate the date and name of a mock participant for a pilot session. 
* Pilot participant email:
* Date and time of pilot session: 

### Research sessions
What dates do you plan to do research? 

### Length of sessions
45 minutes. Perigean will schedule the sessions with buffer 15 minutes of buffer time to allow for participants who can't make it on time, or if you might go over time.

### Availability
When would you like sessions scheduled? Please list exact dates and times in EASTERN Standard Time. *Note: we recommend providing availability outside of work hours, as many Veterans are only available before and after working times, and live across the U.S.* 
Please request enough dates and time slots for the number of requested participants. (e.g. Monday 9-1, 3-6; Tuesday 9-6, etc.). 
	
## Team Roles	

- Moderator: Camille Green ,404-428-2313,Camille.green@oddball.io
- Research guide writing and task development: Camille Green
- Participant recruiting & screening: Perigean	
- Project point of contact: Camille Green
- Participant(s) for pilot test: TBD	
- Note-takers: Perigean, Aubrey Archangel, Sarifa Khalilullah 
- Observers: chante.lantosswett@va.gov,anita.middleton@oddball.io

**Comment by UXCamG** [comment]
Brainstorming session was captured here: https://jamboard.google.com/d/1dXlRa-x8lCjMzGFwQyNCU83_3UotWQ2HAzPqz_AhZXU/edit?usp=sharing

**Comment by UXCamG** [comment]
# Conversation Guide 

## Moderator logistics
- Complete any project-specific setup tasks for this study (List them below)
- [ ] Find Resources and Support 
- [ ] Find 
- [ ] Step 3


### Day of the session
- Use [#feedback-backchannel](https://dsva.slack.com/messages/C40B45NJK/details/) in 
Slack for real-time feedback from observers.
- Mute your Slack notifications.
- Send out the [observer instructions](https://depo-platform-documentation.scrollhelp.site/research-design/Observer-guidelines.1622311177.html) to your observers (Slackbot will do this for you if you type "observer instructions" into [#feedback-backchannel](https://dsva.slack.com/channels/feedback-backchannel)).



### Starting the session

- In the Attendees section, make sure everyone except the participant is on mute.
- In the Participant "more" menu, make sure "Play Join and Leave Sound" is unchecked
- Check your [screensharing setup] and share permissions for participants.
- When the participant is ready, begin the session with the following intro*



## Intro - 5 minutes

Thanks for joining us today! My name is [NAME] and [NOTETAKER NAME] is taking notes.
I also have a couple other colleagues in the Zoom waiting room who would like to observe and take notes. But before we get to them... 

Today we're going to talk about [description of your project].


Before we start, I have a few things that I want to go over with you:

- **This entire session should take about 60 minutes.** I want to be sure not to keep you much longer, so I may occasionally prompt you with the next question or topic.
- **In this session, we want to hear your honest opinions.** There are no right or wrong answers; we are not testing your knowledge or abilities. I welcome any and all of your feedback, and I won't be offended by any opinions you express.
- **Your participation in this interview is completely voluntary.** You do not have to participate if you don't want to. Whether you choose to participate or not will not affect your benefits in any way with the VA. If you would like to stop the session or do not want to answer a question for any reason at any time, that's no problem. Please let me know.
- I know the folks who invited you to participate in this interview asked if it was alright to allow people to observe this interview, but I want to double-check with you before we get going. **Are you okay with a couple of additional observers during this session?**
   - If no: Use Slack to kindly ask the observers to leave the Zoom.
   - If yes: Admit observers.
   - Then, ask the notetaker and observers to give a brief, camera-on introduction, like their first name and position with VA.gov. (This is meant to humanize the interaction of observation for the participant.)
   - Once complete, ask the notetaker and observers to turn off their cameras for the remainder of the session to save bandwidth
- **Are you comfortable if I record my screen and audio as we talk today?** We use the recordings to confirm that we have captured your opinions accurately. After we complete the analysis, the recordings are destroyed, and none of your comments will be attributed to you directly. We either report findings out in groups (i.e., 5 out of 10 participants said this), or if we want to quote you directly, we take away any identifying information and give you a participant ID, like P7.
   - Knowing that, are you comfortable if I record my screen and audio as we talk today?
- **Would you be comfortable if we decided to use a clip of our session with you to help our partners understand Veterans’ experiences?** One of the ways that we share findings with our partners across the VA is by showing short video clips of sessions with Veterans. In these video clips, we blur out any information that could possibly be used to identify you, like your onscreen image. Sharing these video clips can be a compelling way for our partners to better understand what Veterans go through when using VA.gov and our digital products. These clips will not be shared outside of the VA. We don't know at this point if we will or not, but we like to ask ahead of time. At the end of the interview, I'll also confirm that this is okay with you, so you will get a chance to veto this once you know what we've discussed.
   - Knowing this, **would you be comfortable if we decided to use a clip of our session with you to help our partners understand Veteran's experiences?**
   - If yes: **Once I start recording, I am going to confirm that you are okay with me recording this session once more.**

*Start recording.*

- **I have started recording.** I'd like to confirm: Are you comfortable if I record my screen and the audio as we talk today?
- **Are you comfortable if we decide to use a recorded clip from our discussion today to share with other VA partners if we remove any information that can be used to identify you?**


## Warm-up Questions (Optional) - X minutes

Before we look at the website, let’s start with a few warm-up questions.

*Background info about the participant should have been provided by Coordinator already, so use this time to ask any other questions that you feel will help with the session. This warm-up is not required*

## First Task: [TASK NAME] - X minutes

- Step 1
- Step 2

### Things to watch for:

- [Does the participant understand meaning of xyz?]
- [Does the participant notice xyz?]
- [Is the participant aware that they may be sharing personal information on their screen, and are they okay with that? - per question as needed]
- [Etc]

## Second Task: [TASK NAME] - X minutes

- Step 1
- Step 2

### Things to watch for:

- [Does the participant understand meaning of xyz?]
- [Does the participant notice xyz?]
- [Etc]

## Third Task: [TASK NAME] - X minutes

- Step 1
- Step 2

### Things to watch for:

- [Does the participant understand meaning of xyz?]
- [Does the participant notice xyz?]
- [Etc]

## Post-Task Interview - X minutes

- Question 1
- Question 2
- Any questions for me? 

## Consent to use video clips

- I want to circle back and re-confirm that **you are comfortable if we were to share a video clip of this session with our VA partners?** Just to remind you, we will remove all information that could be used to identify you before sharing and will only share the videos with VA partners with the purpose of improving VA.gov and other digital experiences.
   - If yes: Is there any part of our discussion you would NOT want shared in a video clip?


## Thank-You and Closing - X minutes

Well we really appreciate you taking the time to share your thoughts with us today. Your feedback is so helpful to us as we continue to work on the site and make sure it really works for Veterans.

Thanks! Lastly, Perigean will be sending you a thank you note with a little blurb that you can pass along to other Veterans you may know to provide them the chance to participate in future research studies.

Thank you so much again, and enjoy the rest of your day!

##

## Emergency Exit

[In the rare case of an emergency during a session, be prepared with an [emergency exit strategy](https://depo-platform-documentation.scrollhelp.site/research-design/Research-Safety-and-Emergency-Exit-Strategies.2143649793.html#ResearchSafetyandEmergencyExitStrategies-Sampleexitstrategies). Plan to include here at least one option, so its easy to find during a session. Take some time to practice so it feels more comfortable.]

## Sample Conversation Guide _(note: this link points to resources in the va.gov-team GitHub repo)_

- [Veteran ID Card Discovery Interview](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-id-cards/research/discovery/discovery-conversation-guide.md)

**Comment by UXCamG** [comment]
# Conversation Guide 

## Moderator logistics
- Complete any project-specific setup tasks for this study (List them below)
- [ ] Find Resources and Support 
- [ ] Find 
- [ ] Step 3


### Day of the session
- Use [#feedback-backchannel](https://dsva.slack.com/messages/C40B45NJK/details/) in 
Slack for real-time feedback from observers.
- Mute your Slack notifications.
- Send out the [observer instructions](https://depo-platform-documentation.scrollhelp.site/research-design/Observer-guidelines.1622311177.html) to your observers (Slackbot will do this for you if you type "observer instructions" into [#feedback-backchannel](https://dsva.slack.com/channels/feedback-backchannel)).



### Starting the session

- In the Attendees section, make sure everyone except the participant is on mute.
- In the Participant "more" menu, make sure "Play Join and Leave Sound" is unchecked
- Check your [screensharing setup] and share permissions for participants.
- When the participant is ready, begin the session with the following intro*



## Intro - 5 minutes

Thanks for joining us today! My name is [NAME] and [NOTETAKER NAME] is taking notes.
I also have a couple other colleagues in the Zoom waiting room who would like to observe and take notes. But before we get to them... 

Today we're going to talk about [description of your project].


Before we start, I have a few things that I want to go over with you:

- **This entire session should take about 60 minutes.** I want to be sure not to keep you much longer, so I may occasionally prompt you with the next question or topic.
- **In this session, we want to hear your honest opinions.** There are no right or wrong answers; we are not testing your knowledge or abilities. I welcome any and all of your feedback, and I won't be offended by any opinions you express.
- **Your participation in this interview is completely voluntary.** You do not have to participate if you don't want to. Whether you choose to participate or not will not affect your benefits in any way with the VA. If you would like to stop the session or do not want to answer a question for any reason at any time, that's no problem. Please let me know.
- I know the folks who invited you to participate in this interview asked if it was alright to allow people to observe this interview, but I want to double-check with you before we get going. **Are you okay with a couple of additional observers during this session?**
   - If no: Use Slack to kindly ask the observers to leave the Zoom.
   - If yes: Admit observers.
   - Then, ask the notetaker and observers to give a brief, camera-on introduction, like their first name and position with VA.gov. (This is meant to humanize the interaction of observation for the participant.)
   - Once complete, ask the notetaker and observers to turn off their cameras for the remainder of the session to save bandwidth
- **Are you comfortable if I record my screen and audio as we talk today?** We use the recordings to confirm that we have captured your opinions accurately. After we complete the analysis, the recordings are destroyed, and none of your comments will be attributed to you directly. We either report findings out in groups (i.e., 5 out of 10 participants said this), or if we want to quote you directly, we take away any identifying information and give you a participant ID, like P7.
   - Knowing that, are you comfortable if I record my screen and audio as we talk today?
- **Would you be comfortable if we decided to use a clip of our session with you to help our partners understand Veterans’ experiences?** One of the ways that we share findings with our partners across the VA is by showing short video clips of sessions with Veterans. In these video clips, we blur out any information that could possibly be used to identify you, like your onscreen image. Sharing these video clips can be a compelling way for our partners to better understand what Veterans go through when using VA.gov and our digital products. These clips will not be shared outside of the VA. We don't know at this point if we will or not, but we like to ask ahead of time. At the end of the interview, I'll also confirm that this is okay with you, so you will get a chance to veto this once you know what we've discussed.
   - Knowing this, **would you be comfortable if we decided to use a clip of our session with you to help our partners understand Veteran's experiences?**
   - If yes: **Once I start recording, I am going to confirm that you are okay with me recording this session once more.**

*Start recording.*

- **I have started recording.** I'd like to confirm: Are you comfortable if I record my screen and the audio as we talk today?
- **Are you comfortable if we decide to use a recorded clip from our discussion today to share with other VA partners if we remove any information that can be used to identify you?**


## Warm-up Questions (Optional) - X minutes

Before we look at the website, let’s start with a few warm-up questions.

*Background info about the participant should have been provided by Coordinator already, so use this time to ask any other questions that you feel will help with the session. This warm-up is not required*

## First Task: [TASK NAME] - X minutes

- Step 1
- Step 2

### Things to watch for:

- [Does the participant understand meaning of xyz?]
- [Does the participant notice xyz?]
- [Is the participant aware that they may be sharing personal information on their screen, and are they okay with that? - per question as needed]
- [Etc]

## Second Task: [TASK NAME] - X minutes

- Step 1
- Step 2

### Things to watch for:

- [Does the participant understand meaning of xyz?]
- [Does the participant notice xyz?]
- [Etc]

## Third Task: [TASK NAME] - X minutes

- Step 1
- Step 2

### Things to watch for:

- [Does the participant understand meaning of xyz?]
- [Does the participant notice xyz?]
- [Etc]

## Post-Task Interview - X minutes

- Question 1
- Question 2
- Any questions for me? 

## Consent to use video clips

- I want to circle back and re-confirm that **you are comfortable if we were to share a video clip of this session with our VA partners?** Just to remind you, we will remove all information that could be used to identify you before sharing and will only share the videos with VA partners with the purpose of improving VA.gov and other digital experiences.
   - If yes: Is there any part of our discussion you would NOT want shared in a video clip?


## Thank-You and Closing - X minutes

Well we really appreciate you taking the time to share your thoughts with us today. Your feedback is so helpful to us as we continue to work on the site and make sure it really works for Veterans.

Thanks! Lastly, Perigean will be sending you a thank you note with a little blurb that you can pass along to other Veterans you may know to provide them the chance to participate in future research studies.

Thank you so much again, and enjoy the rest of your day!

##

## Emergency Exit

[In the rare case of an emergency during a session, be prepared with an [emergency exit strategy](https://depo-platform-documentation.scrollhelp.site/research-design/Research-Safety-and-Emergency-Exit-Strategies.2143649793.html#ResearchSafetyandEmergencyExitStrategies-Sampleexitstrategies). Plan to include here at least one option, so its easy to find during a session. Take some time to practice so it feels more comfortable.]

## Sample Conversation Guide _(note: this link points to resources in the va.gov-team GitHub repo)_

- [Veteran ID Card Discovery Interview](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-id-cards/research/discovery/discovery-conversation-guide.md)

**Comment by UXCamG** [comment]
# Initiative Brief Template
#### Overview
The Veterans support team would like to suggest an expanded version of the feedback  tool (Good/Bad Rating) and we want to use Medallia to capture the data. Medallia will offer a combined experience for the Veterans as well streamline operations internally. Additionally Medallia already has analytics capabilities that the Veterans support team can build off of. The expanded version of the rating tool that we are interested in utilizing will allow users to provide details for their  ratings on  multiple article pages on R&S.

## Problem
Currently the resources and support article pages have the option for a veteran to submit whether their experience on the page was good or bad. The veterans support team has been discussing additional messages for selection so we can further analyze the reason for a rating (i.e why was it good or bad). When we discussed adding questions for the article experience we have a couple options to explore in regard to its styling. Our options require expanding the rate experiences towards if this then that decision trees, etc...

## Desired User Outcomes
-Users will have a consistent experience when providing feedback
-Rating tool will be located in a consistent part of the page and users will know where to look on a page in order to provide feedback
-Solving the issue of providing detailed feedback could provide less frustration, more efficiency, ability to submit issues, and just an overall improved experience for the Veterans


## Undesired User Outcomes
-Creating a frustrating experience for the Veterans
-Creating a more time consuming experience for the Veterans
-Veterans may be confused why they are asked different questions on different pages on the website note- research to validate
-Inconsistent UI for submitting feedback
-Users feel interrupted and/or frustrated by attempts to collect feedback

## Desired Business Outcomes
-Streamlined sensible process for Veterans to submit feedback
-Product/content owners are able to quickly and easily access user feedback
-Product/content owners are able to understand user feedback in order to make improvements
-Utilization of Medallia since it is already implemented and provides analytics

## Undesired Business Outcomes
-->
-Additional similar processes in different groups
-Additional reporting tool creations
-User feedback is hard to access and/or understand
-The type of feedback collected is not helpful to the product/content owner
---
## Measuring Success
*Utilization of the solution implemented will determine the success of using Medallia vs. the existing good/bad feedback tool.
### Key Performance Indicators (KPIs)
> 💡 *VA.gov products measure success against Ease of use, Service Completion, Trust/Satisfaction, Health.*<sup>1</sup>\
> 💡 *Identify balancing metrics to ensure you're not bringing about undesired outcomes.*

- Product KPI | Baseline | Target | Link to data source (e.g. GA, Domo)

---

## Discovery
### Assumptions
-Medallia can be enabled/disabled on certain pages
-Veterans support could have a separate set of questions or add questions to Medallia and have those presented on the appropriate article pages
-Medallia's Intercept Tool allows users to submit feedback only once and will not reappear until 90 days later. We will not be utilizing this tool.

-Create one combined feedback mechanism for Veterans
-This makes more sense then having good/bad and Medallia because it is confusing to Veterans as to why there are two surveys and also inefficient from a development, maintenance and KPI's perspective
-We have decided to limit the additional feedback responses so that it is quick for the Veterans

Initiatives
Neither a initiative brief or previous product outline exists for good/bad component as it was intended to be an enduring component as it was a stop gap build based on the fact Medallia rolling on to VA.gov and once that was completed our intention was to revert back to a Medallia like CSAT capture

#### Go-to-market 
> Currently the good/bad component rating is available on article pages and the Medallia survey is on the feedback button near the footer on the site. Currently Good/Bad is just that without no supporting information. Medallia has the ability for VA.gov to streamline the survey results to one experience and should be marketed as such.

TBD- This will need to be decided with  Chante Lantos as well as Anita Middleton and  team to ensure a seamless rollout that enhances the user experience. Additionally when rolled out we will want to make sure we are very clear and concise to the Veteran about what they are responding to i.e an article, page. etc..
--- 

## Launch Planning
### Collaboration Cycle

-[ Kickoff ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65297)

### Timeline 
> *Describe any major milestones for this initiative including organizational, legislative, etc. constraints.*

* [Link to Release Plan for this Initiative](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/release-plan-template.md)

#### Initiative Launch Dates
- *Target Launch Date*
  - tbd
- *Actual Launch Date* 
  - tbd

---
   
## Screenshots

### Before

### After

---

#### Communications
*Where will you discuss this initiative?*

<details>

- Team Name: Veterans Support Team
- GitHub Label(s): @clantosswett @UXCamG @ATMiddleton @aubreyarcangel 
- Slack channel: [Slack Channel](https://join.slack.com/share/enQtNTkyMDU0MDIwNzY5Ni1jZTBiMTFjNzRhMTgwZmRiOTgzNWE0ZDc3OGY4NjY4YjZiOGEyMWZmMTE2ZWRlMzc4OTFmZTIxMTMzMGZkMThh)
- Product POCs:

</details>


#### Stakeholders
*What offices/departments are critical to make this initiative successful?*

<details>
  
- Office/Department:
- Contact(s): 
 
</details>

---
<sup>1</sup> [VA.gov Analytics - KPI Framework](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/Analytics%20Playbook/va-gov-platform-analytics-kpi-framework.pdf)\
<sup>2</sup> [SVPG: The Four Big Risks](https://svpg.com/four-big-risks/)

**Comment by UXCamG** [comment]
This thread has drafts of the documents required for the collaboration cycle , design intent, and research. At this time, we are waiting on feedback from the collaboration cycle and Research ops teams.

**Comment by UXCamG** [comment]
This thread has drafts of the documents required for the collaboration cycle , design intent, and research. At this time, we are waiting on feedback from the collaboration cycle and Research ops teams.

**Comment by ATMiddleton** [comment]
Combined with https://app.zenhub.com/workspaces/contact-center-62cdd9546ec1530018209672/issues/gh/department-of-veterans-affairs/va.gov-team/66100

Collab cycle broken into multiple issues.
